app/javascript/components/common/ExerciseIcon.tsx

```diff
@@ -1,7 +1,23 @@
 import React from 'react'
-import { GraphicalIcon } from './GraphicalIcon'
 
-export const ExerciseIcon = ({ icon }: { icon: string }): JSX.Element => {
-  const tmpIconName = icon.replace('sample', 'sample-exercise')
-  return <GraphicalIcon icon={tmpIconName} className="c-exercise-icon" />
+type ExerciseIconProps = {
+  iconUrl: string
+  title?: string
+  className?: string
+}
+
+export function ExerciseIcon({ iconUrl, title, className }: ExerciseIconProps) {
+  let classNames = ['c-icon c-exercise-icon']
+  if (className !== undefined) {
+    classNames.push(className)
+  }
+
+  return (
+    <img
+      className={classNames.join(' ')}
+      src={iconUrl}
+      alt={title ? `Icon for exercise called ${title}` : undefined}
+      role={title ? undefined : 'presentation'}
+    />
+  )
```

Comment:
@kntsoriano All exercise and track icons are now serialized as `icon_url` and rendered through these helpers. 

(They're actually currently going via webpack too at the Rails side, but eventually they'll just be hard-assets in that live in a fixed dir on S3, and are stored in website-icons as in v2. That means we don't have to deploy the website or make code changes to allow new exercises to be added)